app/models/consultation.rb

```diff
@@ -84,6 +85,8 @@ class AddNeighbourAddressesError < StandardError; end
     end
   end
 
+  validates :deadline, presence: true, on: :send_neighbour_emails
```

Comment:
Is this validation context ever used?

Reply:
No — it was but I've broken it in the process of trying to address Celia's feedback 🤦🏻